### PR TITLE
when testplan executed, send messgae with orgID and projectID

### PR DESCRIPTION
**Pull request by ZLValien**
#### What type of this PR

/kind bugfix

#### What this PR does / why we need it:

send messgae with orgID and projectID
eventBox get http path form webHook, if orgID and projectID are -1 ,will insert two httpPath

#### Which issue(s) this PR fixes:

- Fixes #your-issue_number
- [Erda Cloud Issue Link](https://erda.cloud/erda/dop/projects/387/issues/bug?id=222230&issueFilter__urlQuery=eyJzdGF0ZUJlbG9uZ3MiOlsiT1BFTiIsIldPUktJTkciLCJXT05URklYIiwiUkVPUEVOIiwiUkVTT0xWRUQiXSwiYXNzaWduZWVJRHMiOlsiMTAwMDgzMiJdfQ%3D%3D&issueTable__urlQuery=eyJwYWdlTm8iOjEsICJwYWdlU2l6ZSI6MTB9&issueViewGroup__urlQuery=eyJ2YWx1ZSI6InRhYmxlIiwiY2hpbGRyZW5WYWx1ZSI6eyJrYW5iYW4iOiJkZWFkbGluZSJ9fQ%3D%3D&iterationID=429&type=BUG)

#### Specified Reviewers:

/assign @your-reviewer

#### Need cherry-pick to release versions?

/cherry-pick release/1.3

### CODE REVIEW

File: modules/pipeline/aop/plugins/pipeline/testplan_after/plugin.go [hunk 1]
```diff
@@ -188,3 +172,38 @@ func convertReport(pipelineID uint64, report spec.PipelineReport) (ApiReportMeta
...
+	project, err := p.Bundle.GetProject(projectID)
+	if err != nil {
+		return err
+	}
+
+	ev2 := &apistructs.EventCreateRequest{
+		EventHeader: apistructs.EventHeader{
+			Event:         bundle.AutoTestPlanExecuteEvent,
+			Action:        bundle.UpdateAction,
+			OrgID:         fmt.Sprintf("%d", project.OrgID),
```
Comment (Effet) [inline modules/pipeline/aop/plugins/pipeline/testplan_after/plugin.go:194]: why not get org-id from header, why we need query project entity by bundle?

**Review by Effet — CHANGES_REQUESTED** [review]

File: modules/pipeline/aop/plugins/pipeline/testplan_after/plugin.go (hunk 1 above)
Comment (ZLValien) [reply modules/pipeline/aop/plugins/pipeline/testplan_after/plugin.go:194]: image
Pipeline.Labels only hava key:has-report-auto-test-execute-config value:true.
in execute config, org_id is null, only have projectID

Comment (ZLValien) [reply modules/pipeline/aop/plugins/pipeline/testplan_after/plugin.go:194]: I will try to set projectID and orgID when create pipeline

Comment (ZLValien) [reply modules/pipeline/aop/plugins/pipeline/testplan_after/plugin.go:194]: done

File: modules/pipeline/aop/plugins/pipeline/testplan_after/plugin.go [hunk 2]
```diff
@@ -188,3 +170,24 @@ func convertReport(pipelineID uint64, report spec.PipelineReport) (ApiReportMeta
...
 	return meta, nil
 }
+
+func (p *provider) sendMessage(req testplanpb.Content, ctx *aoptypes.TuneContext) error {
+	ev2 := &apistructs.EventCreateRequest{
+		EventHeader: apistructs.EventHeader{
+			Event:         bundle.AutoTestPlanExecuteEvent,
+			Action:        bundle.UpdateAction,
+			OrgID:         ctx.SDK.Pipeline.Labels[apistructs.LabelProjectID],
+			ProjectID:     ctx.SDK.Pipeline.Labels[apistructs.LabelOrgID],
```
Comment (Effet) [inline modules/pipeline/aop/plugins/pipeline/testplan_after/plugin.go:180]: 弄反了

**Review by Effet — CHANGES_REQUESTED** [review]

**Review by Effet — APPROVED** [review]

**Pull request merged**